### PR TITLE
Link redirects to Curam main page

### PR DESCRIPTION
**Pull request by EphraimAssi**
# Description 📝

User name is a link, redirects to Curam main page

## Taiga Link 🔗

https://taiga.dts-stn.com/project/ericdube-hfp-shared-backlog/task/372

## Checklist ✅
- [ ] created story for all visual components
- [ ] created or modified necessary unit tests
- [ ] created or modified e2e tests
- [ ] modified README or any applicable documentation on the changes you made

### CODE REVIEW

File: src/components/molecules/LogInOut.js [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
 import PropTypes from "prop-types";
 import { ActionButton } from "../atoms/ActionButton";
+import { Alink } from "../atoms/Alink";
 
 /**
  * This component show login or a logout button according to the user authenticated flag
  */
 export function LogInOut(props) {
+  //Link for user
+  let url = "https://169.59.166.63:9044/ua/";
```
Comment (Moro-Code) [inline src/components/molecules/LogInOut.js]: never hardcode variables like this...use variable from [variable.js](https://github.com/DTS-STN/ServicePrototypeFrontend/blob/main/src/variables.js). Ideally this should be a prop and should be passed in from the Page component

**Review by Moro-Code — CHANGES_REQUESTED** [review]

File: src/components/molecules/LogInOut.js (hunk 1 above)
Comment (EphraimAssi) [reply src/components/molecules/LogInOut.js]: I changed it now to pass it through props.

Comment (MarcoGoC) [reply src/components/molecules/LogInOut.js]: you may want to use the props directly instead of thru a variable. it is a suggestion only

**Review by will0684 — APPROVED** [review]
Looks good thanks Ephraim

**Pull request closed without merging**